packages/zowe-explorer/src/extension.ts

```diff
@@ -164,6 +164,16 @@ export async function activate(context: vscode.ExtensionContext): Promise<ZoweEx
             }
 
             const creds = await Profiles.getInstance().promptCredentials(profileName, true);
+
+            try {
+                const updatedProfileInfo = Profiles.getInstance().loadNamedProfile(profileName);
+                node.setProfileToChoice(updatedProfileInfo);
```

Comment:
Does the `Profiles.getInstance().allProfiles` array need updated too ?
I'm asking because I remember we discussed something around updating the node and the `allProfiles` array simultaneously.
But if updating the node alone works, that's fine by me 😋

Reply:
I guess I didn't find any strange behavior after updating the node alone.